terms-and-conditions-no.md

```diff
@@ -1,187 +1,267 @@
 # Alminnelige betingelser for Bjerk AS
 
-## 1. Innledning
+## Betingelser
 
-### 1. 1. Betingelser
+Disse generelle avtalevilkår gjelder ved leveranse av tjenester og produkter
+(Leveransen) fra Bjerk AS (Leverandøren), såfremt ikke annet er skriftlig avtalt
+mellom Leverandøren og kunden (Kunden) og/eller regulert i særskilte betingelser
+knyttet til den enkelte del av Leveransen.
 
-Disse vilkår gjelder ved kjøp av tjenester og produkter fra Bjerk AS såfremt
-ikke vilkårene er endret ved skriftlig avtale mellom Bjerk AS og Kunden. Med
-mindre annet er avtalt, er betingelsene gjeldende også ved gjentatte oppdrag for
-Kunden.
-
-### 1. 2. Definisjon
+## Definisjon
 
 Med Bjerk AS menes Bjerk AS med organisasjonsnummer 998 732 867 registrert
-Brønnøysundregistrene. Med Kunden menes i dette dokumentet den person eller
-instans som bestiller en tjeneste eller vare fra Bjerk AS.
+Brønnøysundregistrene. Med «Kunden» menes den person eller instans som bestiller
+en tjeneste eller et produkt fra Leverandøren.
 
-## 2. Oppdraget
+## Informasjon om Leveransen
 
-### 2. 1. Bestilling og ordrens gyldighet
+Kunden skal gis tilgang til dokumentasjon som Leverandøren vanligvis lar følge
+med ved Leveransen.
```

Comment:
Usikker på ordlyden her / relevans. Bjerk vil ordinært sett kun levere tjenester, slik at det _følger ikke med_ noe i en leveranse sånn uten videre.

Reply:
"Vanligvis" er kanskje et forvirrende ord å bruke. Vanligvis for Bjerk eller vanligvis for andre selskaper? Hva om kunden har en egen forventning utifra spesielle rutiner hos tidligere leverandører som påvirkere deres perskeptiv på hva "vanligvis" er?